hosts/dinar-ephemeral-alpha/default.nix

```diff
@@ -71,8 +72,8 @@ in
     enable = true;
     settings.PasswordAuthentication = false;
     hostKeys = [{
-      path = "/mnt/secrets/ssh/id_ed25519";
-      type = "ed25519";
+      path = sshKeysPath;
+      type = "id_ed25519";
```

Comment:
this could alternatively be configured to be the age.sshKeyPaths like above, would probably be less error-prone that way

Reply:
Can't figure out how one would do this

ref: https://github.com/NixOS/nixpkgs/blob/f5364316e314436f6b9c8fd50592b18920ab18f9/nixos/modules/services/networking/ssh/sshd.nix#L229